modules/monitoring/manifests/checks/rds.pp

```diff
@@ -53,6 +53,8 @@
       monitoring::checks::rds_config { $servers:
         region => $region,
       }
-      Monitoring::Checks::Rds_config <<| |>>
+      Monitoring::Checks::Rds_config <<| |>> {
+        region => $region,
+      }
```

Comment:
This seems very similar to the section above 🤔

Reply:
yah, so I think lines 53-55 create a bunch of resources depending on whats in hieradata ($servers). Then 56-58 collects any other monitoring resources made else where in puppet manifests.